services/bundle_analysis/report.py

```diff
@@ -372,50 +374,51 @@ def save_measurements(self, commit: Commit, upload: Upload) -> ProcessingResult:
             ]
 
             db_session = commit.get_db_session()
```

Comment:
The diff is basically from:
```
for bundle_report in bundle_analysis_report.bundle_reports():
    <all the saving code>
```
to
```
bundle_report = bundle_analysis_report.bundle_report(bundle_name)
<all the saving code>
```

Reply:
Thank you for this comment, most helpful